src/components/BMDashboard/Issue/Issue.jsx

```diff
@@ -0,0 +1,347 @@
+            </Col>
+            <Col>
+              <Input
+                className="issue-form-override-css issue-date-input"
+                type="date"
+                name="issueDate"
+                id="issueDate"
+                onChange={e => setFormData({ ...formData, issueDate: e.target.value })}
+              />
+            </Col>
+          </Row>
+        </FormGroup>
+        <Row>
+          <Col>
+            <Label className="red-text">{NOTE}</Label>
+          </Col>
+        </Row>
+        <FormGroup>
+          <Row>
+            <Col>
+              <Label className="sub-title-text">{ISSUE_TYPE}</Label>
+            </Col>
+          </Row>
+          <Row>
+            <Col>
+              <Input
+                className="issue-form-override-css"
+                type="select"
+                name="dropdown"
+                value={formData.dropdown}
+                onChange={e => handleDropdownChange(e.target.value)}
+              >
+                {dropdownOptions.map(option => (
+                  <option key={option} value={option}>
+                    {option}
+                  </option>
+                ))}
+              </Input>
+            </Col>
+          </Row>
+        </FormGroup>
+        <FormGroup>
+          <Row>
+            <Col>
+              <Label className="sub-title-text">{CONSEQUENCES_TITLE}</Label>
+            </Col>
+          </Row>
+          {checkboxOptions.map(pair => (
+            <Row key={pair} className="consequences-checkboxes">
+              {pair.map(option => (
+                <Col key={option}>
+                  <Label check>
+                    <Input
+                      type="checkbox"
+                      checked={formData.checkboxes.has(option) || false}
+                      onChange={() => handleCheckboxChange(option)}
+                    />
+                    {option}
+                  </Label>
+                  {option === 'Other' && (
+                    <Input
+                      className="issue-form-override-css"
+                      type="textarea"
+                      rows="1"
+                      placeholder="If other is selected, please specify."
+                      value={formData.other}
+                      onChange={e => handleOtherInputChange(e, option)}
+                    />
+                  )}
+                </Col>
+              ))}
+            </Row>
+          ))}
+        </FormGroup>
+        <FormGroup>
+          <Row>
+            <Col>
+              <Label className="sub-title-text">{RESOLVED}</Label>
+            </Col>
+          </Row>
+          <Row className="issue-radio-buttons">
+            <Col className="issue-radio-buttons">
+              <Label check>
+                <Input
+                  type="radio"
+                  name="radioOption"
+                  onChange={() => setFormData({ ...formData, resolved: 'Yes' })}
+                />
+                Yes
```

Comment:
Could we please change the cursor to a pointer when hovering over Yes or no text ? This will improve the user experience by making it clear that the text is also clickable.